packages/manager/apps/vrack-services/src/pages/dashboard/[id]/CreateSubnet.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useQueryClient, useMutation } from '@tanstack/react-query';
+import {
+  ODS_INPUT_SIZE,
+  ODS_INPUT_TYPE,
+  OdsInputValueChangeEvent,
+} from '@ovhcloud/ods-components/input';
+import { OsdsInput } from '@ovhcloud/ods-components/input/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import i18next from 'i18next';
+import {
+  OsdsRadio,
+  OsdsRadioGroup,
+} from '@ovhcloud/ods-components/radio/react';
+import { OdsRadioGroupValueChangeEvent } from '@ovhcloud/ods-components/radio';
+import { OsdsRadioButton } from '@ovhcloud/ods-components/radio-button/react';
+import { ODS_RADIO_BUTTON_SIZE } from '@ovhcloud/ods-components/radio-button';
+import {
+  getListingIcebergQueryKey,
+  updateVrackServices,
+  updateVrackServicesQueryKey,
+} from '@/api';
+import { CreatePageLayout } from '@/components/layout-helpers';
+import {
+  displayNameInputName,
+  cidrInputName,
+  serviceRangeSelectName,
+  vlanInputName,
+  vlanNumberInputName,
+  noVlanOptionValue,
+  vlanNumberOptionValue,
+  defaultCidr,
+  defaultServiceRange,
+} from './constants';
+import { useVrackService } from './utils';
+import { ApiError, ErrorPage } from '@/components/Error';
+import { FormField } from '@/components/FormField';
+
+export const isValidVlanNumber = (vlan: number) => vlan >= 2 && vlan <= 4094;
+
+export function breadcrumb() {
+  return i18next.t('vrack-services/subnets:createPageTitle');
+}
+
+const SubnetCreationPage: React.FC = () => {
+  const { t } = useTranslation('vrack-services/subnets');
+  const { id } = useParams();
+  const [displayName, setDisplayName] = React.useState<string | undefined>(
```

Comment:
Sometime you use `React.useState` and in other file you use `{ useState }` could you uniformize import 

Prefer importing `{ useState }`

Reply:
The only files where it's not React.useState are the ones generated
React.useState is better since we already import React so why complexify the import ?

Reply:
fixed here https://github.com/ovh/manager/pull/10510